cmplibrary/src/main/java/com/sourcepoint/gdpr_cmplibrary/GDPRConsentLib.java

```diff
@@ -354,32 +390,58 @@ public void onFailure(ConsentLibException exception) {
         });
     }
 
-    void showView(View view) {
+    void showView(View view, boolean isFromPM) {
         mCountDownTimer.cancel();
         if (!hasParent(view)) {
             runOnLiveActivityUIThread(() -> GDPRConsentLib.this.onConsentUIReady.run(view));
         }
+        if (isFromPM) runPMWillShow();
```

Comment:
could try make a ternary inline check here:
```Java
isFromPM ? runPMWillShow() : runMessageWillShow();
```

Reply:
Also, I don't think this code should be part of this method. It makes forces you to pass this parameter 'isFromPM' all the way down. Instead you could could just call either runPMWillShow() or runMessageWillShow() when needed. Does it make sense to you?

Reply:
We need this code here only as we need to have these callbacks after onConsentUIReady. as per description in ticket.

Reply:
> could try make a ternary inline check here:
> 
> ```java
> isFromPM ? runPMWillShow() : runMessageWillShow();
> ```

We can only perform inline ternary operation with assignment. 
Here we are not assigning value to any variable also these are void methods